tiledb/sm/array_schema/array_schema.h

```diff
@@ -84,6 +84,11 @@ class ArraySchema {
   /*               API                 */
   /* ********************************* */
 
+  /** Returns true if the array allows coordinate duplicates. Applicable
```

Comment:
For consistency with other block comments in this file, put this sentence on the next line and leave this line as `/**`.